### DISCUSSION TITLE
Unable to calculate final grade

### DISCUSSION THREAD

**Discussion by prs3q** [community/community#88791] [Codespaces]
### Select Topic Area

General

### Body

I hope I am posting this in the right way. Basically every time I try to calculate my grade in codespaces, it takes several minutes and then ultimately says “an issue occurred when calculating your final grade”. If anyone has experienced a similar issue and can let me know how to remedy it, that would be greatly appreciated.

**Comment by georgetoloraia** [comment]
1. Update Your Tools:
- Ensure that your tools and dependencies are up to date. Sometimes, an issue has been fixed in a newer version of the software.

2. Check Logs and Error Messages:
- If there are logs or error messages, review them for clues. They can often point you directly to the issue.

3. Isolate the Problem:
- Try to isolate where the problem is occurring. If you have multiple parts in your grade calculation, try running them separately to see if a specific part is causing the issue.

**Reply by DevClaros** [reply]
I would also try installing debugging extensions to help you look at what parts of the assignments might not be running properly and also try running them separately..

**Comment by BraydenC555** [comment]
Been facing this issue for a week now, all of my deadlines are approaching and I dont know what to do. I've been troubleshooting for hours a day doing EVERYTHING I can. Any fixes yet????

**Comment by Judah1035** [comment]
Same exact issue, SOMEBODY GIVE ME AN ANSWER 🙏🙏🙏

**Comment by bundyem2** [comment]
Does anyone know if this has been fixed? I’m having the same issue in January 2026.
